components/org.wso2.carbon.identity.recovery/src/test/java/org/wso2/carbon/identity/recovery/connector/AdminForcedPasswordResetConfigImplTest.java

```diff
@@ -0,0 +1,181 @@
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.recovery.connector;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.governance.IdentityGovernanceException;
+import org.wso2.carbon.identity.recovery.IdentityRecoveryConstants;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import static org.testng.Assert.assertEquals;
+
+/***
+ *This class does unit test coverage for AdminForcedPasswordResetConfigImpl class
+ */
+public class AdminForcedPasswordResetConfigImplTest {
+
+    private AdminForcedPasswordResetConfigImpl adminForcedPasswordResetConfigIml;
+
+    @BeforeTest
+    public void init() {
+        adminForcedPasswordResetConfigIml = new AdminForcedPasswordResetConfigImpl();
+    }
+
+    @Test
+    public void testGetName() throws IdentityGovernanceException {
+        assertEquals("admin-forced-password-reset", adminForcedPasswordResetConfigIml.getName());
+    }
+
+    @Test
+    public void testGetFriendlyName() throws IdentityGovernanceException {
+        assertEquals("Password Reset", adminForcedPasswordResetConfigIml.getFriendlyName());
+    }
+
+    @Test
+    public void testGetCategory() throws IdentityGovernanceException {
+        assertEquals("Account Management Policies", adminForcedPasswordResetConfigIml.getCategory());
+    }
+
+    @Test
+    public void testGetSubCategory() throws IdentityGovernanceException {
+        assertEquals("DEFAULT", adminForcedPasswordResetConfigIml.getSubCategory());
+    }
+
+    @Test
+    public void testGetOrder() throws IdentityGovernanceException {
+        assertEquals(0, adminForcedPasswordResetConfigIml.getOrder());
+    }
+
+    @Test
+    public void testGetPropertyNameMapping() throws IdentityGovernanceException {
+        Map<String, String> nameMapping = new HashMap<>();
+        nameMapping.put(IdentityRecoveryConstants.ConnectorConfig.ENABLE_ADMIN_PASSWORD_RESET_WITH_RECOVERY_LINK,
+                "Enable Password Reset via Recovery Email");
+        nameMapping.put(IdentityRecoveryConstants.ConnectorConfig.ENABLE_ADMIN_PASSWORD_RESET_WITH_OTP,
+                "Enable Password Reset via OTP");
+        nameMapping.put(IdentityRecoveryConstants.ConnectorConfig.ENABLE_ADMIN_PASSWORD_RESET_OFFLINE,
+                "Enable Password Reset Offline");
+
+        Map<String, String> nameMappingExpected = adminForcedPasswordResetConfigIml.getPropertyNameMapping();
+        assertEquals(nameMapping, nameMappingExpected, "Maps are equal");
```

Comment:
3rd parameter in assertEquals method is the message shown in case of assert fail. So it should give the reason for failure. In this case assert will fail if maps are not equal. So the message should rather say "Maps are not equal". Fix all the messages in the PR.